strawberry/django/test/client.py

```diff
@@ -9,7 +9,7 @@ def request(
         body: Dict[str, object],
         headers: Optional[Dict[str, object]] = None,
         files: Optional[Dict[str, object]] = None,
-    ):
+    ) -> Any:
```

Comment:
we should probably type these better in future, since we roughly know what we'll be getting